resqpy/rq_import/add_ab_properties.py

```diff
@@ -0,0 +1,89 @@
+"""add_ab_properties.py: Module to add binary grid properties to an existing RESQMl grid object."""
+
+version = '15th November 2021'
+
+import logging
+
+log = logging.getLogger(__name__)
+log.debug('add_ab_properties.py version ' + version)
+
+import resqpy.grid as grr
+import resqpy.model as rq
+import resqpy.olio.uuid as bu
+import resqpy.olio.xml_et as rqet
+import resqpy.property as rp
+
+
+def add_ab_properties(
+    epc_file,  # existing resqml model
+    grid_uuid = None,  # optional grid uuid, required if more than one grid in model; todo: handle list of grids?
+    ext_uuid = None,  # if None, hdf5 file holding grid geometry will be used
+    ab_property_list = None
+):  # list of (file_name, keyword, property_kind, facet_type, facet, uom, time_index, null_value,
+    #          discrete, realization)
+    """Process a list of pure binary property array files.
```

Comment:
Let's move the argument comments into the docstring.

Reply:
Adding these to the checklist for rq_import, to be completed when further refactoring of add_ab_properties and import_vdb... commences...